### DISCUSSION TITLE
Kubernetes Helm-Chart

### DISCUSSION THREAD

**Discussion by h0jeZvgoxFepBQ2C** [rustdesk/rustdesk#8384] [General]
Hi,

would be really cool if you could offer an official kubernetes helm chart for deployment?

Thank you!

**Comment by RLutsch** [comment]
Should we just build one? I jusst need the implimentation

**Comment by jigassa** [comment]
How difficult can it be?!

**Comment by KlavsKlavsen** [comment]
I found this https://github.com/oslo254804746/rustdesk-server-helm - no license given in repo though - but one could ask author?
